cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImpl.java

```diff
@@ -90,6 +96,65 @@ public void persist(String orgName, String pkgName, String resName, String recor
             insert(resId, records);
         } // if else
     } // persist
+
+    private String resourceLookupOrCreateDynamicFields(String orgName, String pkgName, String resName, String records)
```

Comment:
JavaDoc missing.

Probably an important thing to mention in that JavaDoc is that this method is used by the cygnus-ngis-ld bundle.

Reply:
Added